tensorboard/components/tf_dashboard_common/dashboard-style.html

```diff
@@ -66,7 +82,6 @@
 
       .sidebar-section:last-of-type {
         flex-grow: 1;
```

Comment:
Now that .runs-selector has `flex-grow: 1`, can we drop this `last-of-type` rule?